courses/free-courses-en.md

```diff
@@ -274,6 +275,7 @@
 * [Data Analysis with R](https://www.udacity.com/course/data-analysis-with-r--ud651) - Facebook (Udacity)
 * [Data Cleaning by Rachael Tatman at Kaggle](https://www.kaggle.com/learn/data-cleaning)
 * [Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/dataminingwithweka/)
+* [Data science for beginners](https://github.com/microsoft/Data-Science-For-Beginners) - Microsoft 
```

Comment:
Would be better the front-matter?
> https://microsoft.github.io/Data-Science-For-Beginners

Reply:
Yes that would be more learner friendly interface.

Reply:
Resolved at https://github.com/EbookFoundation/free-programming-books/commit/1ba26e533ca56800b4d2c91674b15fd67f23792d#diff-b0eb8fcca67d6470c73302a9fa829288753e4a1a31ef6815df89165d8676dcd2R278